mcc_generated_files/adcc.c

```diff
@@ -138,12 +138,12 @@ void ADCC_Initialize(void)
     // ADRPT 0; 
     ADRPT = 0x00;
     // ADPCH ANC4; 
-    ADPCH = 0b010100;//x00;
+    ADPCH = 0b111111;//0b010100;//x00;
```

Comment:
wtf is happening here

Reply:
i could change that back but i dont think it makes a real difference

Reply:
I think its fine if you don't comment the lines, but then get rid of the commented-out code